src/screens/Home/details.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import BlankSlate from 'screens/Home/blankSlate';
+
+const Details = () => {
+  const date = new Date();
+  const date_options = { weekday: 'long', month: 'long', day: 'numeric' };
+  const today = date.toLocaleString('en-EN', date_options).toUpperCase();
+
+  return (
+    <div className="home-detail">
+      <p className="home-detail__date">{ today }</p>
+      <h1 className="home-detail__day">Today</h1>
```

Comment:
@rafayet-monon Here too 🙌
I think we can use just a simple tag and styling that with css